CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Layotto Contributor Guide
```

Comment:
Do we still need this file when we already have  docs/zh/development/CONTRIBUTING.md  and docs/en/development/CONTRIBUTING.md ?
I prefer not to add this one because keeping 2 copies up-to-date is easier than 3 :)

Reply:
<img width="1470" alt="image" src="https://user-images.githubusercontent.com/48784001/167259449-22b05d5d-7cff-4cba-bea3-63f7cef69dad.png">

CONTRIBUTING.md is a special file in GitHub Repo, which it related to the community standards.

Reply:
Ok.